### ISSUE TITLE
Agda filetype non-detected/non-existing

### ISSUE THREAD

**Issue by foxyseta** [tree-sitter/tree-sitter-agda#21]
Whenever I open an `.agda`/`.agda-lib` file, the `agda` filetype is not detected, and hence the tree-sitter-agda (which is correctly installed) is not enabled on the buffer.

If I use `:set filetype agda`, the `agda` filetype seems to be never have been defined.

If I manually enable tree-sitter-agda via `:TSEnable`, nothing happens (no syntax highlighting).

**Comment by eldritch-cookie** [comment]
don't know if this response is still useful given it comes 2 years later, but filetype detection isn't done by treesitter but rather vim/neovim and they don't have agda support by default you need to either add the filetype detection yourself or use a plugin that does like [cornelis](https://github.com/agda/cornelis)

**Comment by foxyseta** [comment]
You are totally right. I believe I looked into how the various moving parts of the ecosystem interact shortly after opening the issue, and then forgot to close it. Sorry, and thanks for the correct answer still!
